src/Stripe.net/Entities/Charges/ChargePaymentMethodDetails/ChargePaymentMethodDetailsInteracPresent.cs

```diff
@@ -0,0 +1,106 @@
+
+        /// <summary>
+        /// Authorization response cryptogram.
+        /// </summary>
+        [JsonProperty("emv_auth_data")]
+        public string EmvAuthData { get; set; }
+
+        /// <summary>
+        /// Two-digit number representing the card's expiration month.
+        /// </summary>
+        [JsonProperty("exp_month")]
+        public long ExpMonth { get; set; }
+
+        /// <summary>
+        /// Four-digit number representing the card's expiration year.
+        /// </summary>
+        [JsonProperty("exp_year")]
+        public long ExpYear { get; set; }
+
+        /// <summary>
+        /// Uniquely identifies this particular card number.
+        /// You can use this attribute to check whether two customers
+        /// who've signed up with you are using the same card number,
+        /// for example. For payment methods that tokenize card
+        /// information (Apple Pay, Google Pay), the tokenized number
+        /// might be provided instead of the underlying card number.
+        /// </summary>
+        [JsonProperty("fingerprint")]
+        public string Fingerprint { get; set; }
+
+        /// <summary>
+        /// Card funding type. Can be <c>credit</c>, <c>debit</c>,
+        /// <c>prepaid</c>, or <c>unknown</c>.
+        /// </summary>
+        [JsonProperty("funding")]
+        public string Funding { get; set; }
+
+        /// <summary>
+        /// ID of a card PaymentMethod generated from the card_present
+        /// PaymentMethod that may be attached to a Customer for future
+        /// transactions. Only present if it was possible to generate a
+        /// card PaymentMethod.
+        /// </summary>
+        [JsonProperty("generated_card")]
+        public string GeneratedCard { get; set; }
+
+        /// <summary>
+        /// The last four digits of the card.
+        /// </summary>
+        [JsonProperty("last4")]
+        public string Last4 { get; set; }
+
+        /// <summary>
+        /// Identifies which network this charge was processed on.
+        /// Can be <c>amex</c>, <c>diners</c>, <c>discover</c>,
+        /// <c>interac</c>, <c>jcb</c>, <c>mastercard</c>,
+        /// <c>unionpay</c>, <c>visa</c>, or <c>unknown</c>.
+        /// </summary>
+        [JsonProperty("network")]
+        public string Network { get; set; }
+
+        /// <summary> How were card details read in this transaction.  Can be
+        /// <c>contact_emv</c>, <c>contactless_emv</c>,
+        /// <c>magnetic_stripe_fallback</c>, <c>magnetic_stripe_track2</c>, or
+        /// <c>contactless_magstripe_mode</c>
+        /// </summary>
+        [JsonProperty("read_method")]
+        public string ReadMethod { get; set; }
+
+        /// <summary>
+        /// A collection of fields required to be displayed on receipts.
+        /// Only required for EMV transactions.
+        /// </summary>
+        [JsonProperty("receipt")]
+        public Receipt Receipt { get; set; }
```

Comment:
This should be `ChargePaymentMethodDetailsInteracPresentReceipt` right?

Reply:
yep great catch